modular_citadel/code/game/machinery/vending.dm

```diff
@@ -0,0 +1,3 @@
+/obj/machinery/vending/security
+	contraband = list(/obj/item/clothing/glasses/sunglasses = 2, /obj/item/storage/fancy/donut_box = 2, /obj/item/device/ssword_kit = 1)
```

Comment:
Pretty sure you can do
`contraband[/obj/item/device/ssword_kit] = 1`
to avoid redefining the entire list

Same with premium

Reply:
Doing that exactly I get this error:

`modular_citadel\code\game\machinery\vending.dm:2:error: contraband: left-hand side must be an object variable`

`modular_citadel\code\game\machinery\vending.dm:3:error: premium: left-hand side must be an object variable`